script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt

```diff
@@ -256,12 +290,12 @@ Well, it's fine if you haven't. I expected nothing more from you."
 [sha:73b4fd54311e3753e9e0407ba7b0a841f1a2ff82]{
 -- Page 47, Offset 39487.
 -- 耳元で囁かれる、落ち着き払った男の声。
-A calm man's voice whispers in my ears.
+The calm voice of a man whispers in my ear.
 }
 [sha:1969ee3f469b177afbddf60a106bf2f5fb4d7e27]{
 -- Page 47, Offset 39488.
 -- それは俺の内側―――肉体から生じる、俺ではない他人の声だ。
-It is the voice of another man coming from my insides―――from within my own body.
+It is the voice of someone else coming from within me――from inside my own body.
 }
 [sha:31b12a751e7d27b483788b3991ffccbd7478ff7d]{
 -- Page 48, Offset 39489.
```

Comment:
Reasoning?

Reply:
"You're... Roa!", because of 39491.
Not sure if that's entirely the intended effect in JP, but it seems more of a thing you'd at least say, rather than a straight up "You". You what? It's you? You are? It's either that or some more dramatic change